src/app/components/mno-pagination/mno-pagination.html

```diff
@@ -9,7 +9,7 @@
 </div>
 
 <ul uib-pagination ng-model="page" 
-  ng-show="totalItems > nbItems" 
```

Comment:
This line was preventing the pagination to be displayed if not needed, `totalItems > 0` will always display it?